packages/grid/x-data-grid-pro/src/components/DataGridProColumnHeaders.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { unstable_composeClasses as composeClasses } from '@mui/material';
 import { styled, alpha } from '@mui/material/styles';
+import { useTheme } from '@mui/system';
```

Comment:
A note that the codebase never imported modules from this package so far (only types). It's not necessarily a bad thing to do this, but if there isn't any specific benefit, to change.

Reply:
should I expose API in Datagrid so the user can input direction manually? 

```javascript
<DataGrid 
     ....
    direction="rtl"
/>
```

Reply:
We have imported `useTheme` from `@mui/material/styles`, we can keep doing that for consistency:
https://github.com/mui/mui-x/blob/7a7977fc8e51abefc47d88998e7c00d714df234c/packages/grid/x-data-grid-pro/src/hooks/features/columnResize/useGridColumnResize.tsx#L20